dapps/react-dapp-v2/src/components/Asset.tsx

```diff
@@ -48,11 +48,23 @@ function getAssetIcon(asset: AssetData): JSX.Element {
       return <Icon src={maticLogo} />;
     case "kda":
       return <Icon src={kadenaLogo} />;
+    case "ℏ":
+      return <Icon src={"/assets/hedera-hbar-logo.png"} />;
     default:
       return <Icon src={"/assets/eth20.svg"} />;
   }
 }
 
+function formatAssetBalance(asset: AssetData) {
+  if (!asset.balance) return "0";
+  switch (asset.name.toLowerCase()) {
+    case "ether":
+      return fromWad(asset.balance);
+    default:
+      return asset.balance;
+  }
+}
+
```

Comment:
there may be other assets besides ether that were previously run through this function?  https://github.com/hgraph-io/web-examples/blob/00cf4c7952626894c185a7ba2b3df0d581723f82/dapps/react-dapp-v2/src/helpers/utilities.ts#L201C1-L203C3

```suggestion
function formatAssetBalance(asset: AssetData) {
  if (!asset.balance) return "0";
  switch (asset.name.toLowerCase()) {
    case "hbar":
       return asset.balance;
    default:
      return fromWad(asset.balance);
  }
}

```

Reply:
Done. Just realized you used the "suggested change" feature, after I had already pushed the change 🤦